install_ffmpeg.sh

```diff
@@ -103,7 +103,7 @@ EXTRA_LDFLAGS=""
 if [ $(uname) == "Linux" ]; then
   if [ -e /usr/local/cuda/include ]; then
```

Comment:
This isn't a good check anymore, as `clang` doesn't require CUDA as a build-time dependency. I went with

```
if which clang > /dev/null; then
```

Really what we want to check for is clang >= 8.0, but I dunno how to do that in shell.